NameLayerMain/src/vg/civcraft/mc/namelayer/group/groups/BlackList.java

```diff
@@ -1,11 +1,96 @@
 package vg.civcraft.mc.namelayer.group.groups;
 
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 
-/*
- * Currently wont be implemented.  
- * TTk2 has expressed that he doesnt want this.
- * May come back later to add as an additional feature.
- */
-public class BlackList {
+import vg.civcraft.mc.namelayer.NameLayerPlugin;
+import vg.civcraft.mc.namelayer.group.Group;
 
+public class BlackList {
```

Comment:
Because it's tied into the enum and some other stuff, it would be cleaner to just remove the class. But honestly I really can't think of good arguments against this and don't see why this would be opposed.